pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/db/dao/AnswerDao.java

```diff
@@ -154,11 +154,43 @@ private void createAnswerFileValue(long answerId, FileAnswer answer) {
         if (answer.getValue() == null) {
             return;
         }
-        List<Long> uploadIds = answer.getValue().stream().map(FileInfo::getUploadId).collect(Collectors.toList());
-        int[] inserted = getAnswerSql().bulkInsertFileValue(answerId, uploadIds);
+
+        final List<Long> uploadIds = answer.getValue().stream().map(FileInfo::getUploadId).collect(Collectors.toList());
+        final int[] inserted = getAnswerSql().bulkInsertFileValue(answerId, uploadIds);
         if (inserted.length != uploadIds.size()) {
-            throw new DaoException("Not all file uploads were assigned to answer " + answerId);
+            final var message = String.format("Failed to insert uploads ids for [answer:%s] (%s inserted, %s expected)",
```

Comment:
Not a comment, just want to know your opinion: why do you extracted the message? It's a matter of taste I would say, but do you think this is less readable:

```
  throw new DaoException(String.format("Failed to insert uploads ids for [answer:%s] (%s inserted, %s expected)",
                    answerId,
                    inserted.length,
                    uploadIds.size()));
```

Reply:
Agreed- that is an artifact from a prior change which extended the line past the 140c limit, and was just never changed back.